src/main/java/edu/cornell/kfs/pdp/service/impl/CuPaymentFileServiceImpl.java

```diff
@@ -87,6 +96,55 @@ public void loadPayments(PaymentFileLoad paymentFile, LoadPaymentStatus status,
         status.setLoadStatus(LoadPaymentStatus.LoadStatus.SUCCESS);
     }
     
+    private void setPaymentAddressFieldsForConcurPayments(PaymentFileLoad paymentFile, PaymentGroup paymentGroup) {
+        LOG.debug("setPaymentAddressFieldsForConcurPayments, entering");
+        if (isConcurCustomer(paymentFile)) {
+            Person employee = buildPerson(paymentGroup.getPayeeId());
+            if (ObjectUtils.isNotNull(employee)) {
+                LOG.debug("setPaymentAddressFieldsForConcurPayments, found a concur customer, for employee: " + employee.getName());
+                paymentGroup.setLine1Address(employee.getAddressLine1Unmasked());
+                paymentGroup.setLine2Address(employee.getAddressLine2Unmasked());
+                paymentGroup.setLine3Address(employee.getAddressLine3Unmasked());
+                paymentGroup.setCity(employee.getAddressCityUnmasked());
+                paymentGroup.setState(employee.getAddressStateProvinceCodeUnmasked());
+                paymentGroup.setZipCd(employee.getAddressPostalCodeUnmasked());
+                paymentGroup.setCountry(employee.getAddressCountryCodeUnmasked());
+            } else {
+                LOG.error("setPaymentAddressFieldsForConcurPayments, unable to get a person from the employee ID: " + paymentGroup.getPayeeId());
```

Comment:
We validate employee ID at creation of the PDP XML, so this should never happen, we should probably fail as there is probably something wrong.